index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"/>
+    <link rel="stylesheet" href="style.css">
+    <title>Leo's Portfolio</title>
+</head>
+<body>
+    <nav>
+        <div><a href="index.html">Leo</a></div>
+        <div><a href="#"><i class="fa-solid fa-bars"></i></a></div>
+    </nav>
+    <div class="nav-items">
+        <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#"><i class="fa-regular fa-envelope"></i></a></li>
+        </ul>
+    </div>
+    <!--Home Section-->
+    <div class="home">
+        <div class="intro">
+            <span>Hey There. I'm Leo</span>
+        </div>
+        <div class="job">
+            <span>I'am a full-stack Web developer</span>
+        </div>
+        <div class="bio">
+            I can help your business improve by building a nice website or portfolio. You can check some of my recent works and if you have something to ask me don't hestiate to ask me. Hope we can work together. 
+        </div>
+        <div class="socials">
+            <a href=""><i class="fa-brands fa-github"></i></a>
+            <a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+            <a href=""><i class="fa-brands fa-twitter"></i></a>
+            <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+        </div>
```

Comment:
- [ ] Good job adding your icons @popey17, but The list of icons better be wrapped inside a `ul` tag to group them, and `li` per icon; then each icon has a link (`a` tag). 
- [Optional] Also I noticed you missed the icons for the `medium` and `angelist` icons, Please adding those icons and removing the `facebook` icon.